.github/ISSUE_TEMPLATE/documentation.yml

```
@@ -0,0 +1,30 @@
+name: Documentation
+description: Report problems and suggest ideas for the documentation
+labels: [docs]
+
+body:
+  - type: markdown
+    attributes:
+      value: Before opening a new issue, please search for duplicate or closed issues.
```

I wonder if we can include a link here to make it a bit easier to find out if the issue already exists by including a link, maybe something like (not sure if the link will work like this):

```suggestion
      value: |
        Before opening a new issue, make sure it isn't covered by an existing issue. Please
        search for [issues with the `docs` label][docs-issues] (including closed issues) before
        you continue.
        
        [docs-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Adocs+is%3Aopen)
```

---

This is a good idea, but the link searches only the opened issues. Do you want that to search closed issues too?

---

> This is a good idea, but the link searches only the opened issues. Do you want that to search closed issues too? Also should this link be inline just to match the formatting of the other blocks?

It depends a bit on how much trust you have that the issueing person understand the GitHub UI and is aware of Open v. Closed isues as well as your personally preference. I personally prefer it too link to the open issues and look through open and closed issues separately. Choose what you think's best.


> One more thing, here's a good resource on yaml multiline strings that helped me when making the forms. We want to use >- here to avoid creating newlines. | will keep the line breaks.

Thanks, always get confused by those so just went with some symbol hoping you'd fix it for me :upside_down_face:

---

Keeping links as searching only open issues by default